src/about-us.md

```diff
@@ -2,7 +2,39 @@
 
 Renovate was created by [WhiteSource](https://www.whitesourcesoftware.com/) staff and they continue to work on Renovate.
 
-More than 500 outside contributors helped improve Renovate.
+More than 600 outside contributors helped improve Renovate.
+
+## Special thanks
```

Comment:
https://github.com/renovatebot/renovate/blob/768e178419437a98f5ce4996bafd23f169e530b4/package.json#L81-L120

Do we need to thank some people from our list of `contributors` in our `package.json` on this new page as well?